apps/src/templates/sectionsRefresh/sections-refresh.module.scss

```diff
@@ -55,6 +55,7 @@
 .headerRow {
   background-color: $brand_primary_default;
   color: $white;
+  height: 42px;
```

Comment:
This makes sure that even though the HOC and PL tables wont have a table name at the top, the blue bar will still be the correct height.